src/libraries/Microsoft.Extensions.Logging.Abstractions/tests/Microsoft.Extensions.Logging.Generators.Tests/Microsoft.Extensions.Logging.Generators.Roslyn4.0.Tests.csproj

```diff
@@ -5,7 +5,6 @@
     <DefineConstants>$(DefineConstants);ROSLYN4_0_OR_GREATER</DefineConstants>
     <IsHighAotMemoryUsageTest>true</IsHighAotMemoryUsageTest>
     <EmccLinkOptimizationFlag Condition="'$(ContinuousIntegrationBuild)' == 'true'">-O1</EmccLinkOptimizationFlag>
-    <WasmNativeStrip>false</WasmNativeStrip>
```

Comment:
Setting `EmccLinkOptimizationFlag` will be noop without `WasmNativeStrip` (=> `WasmBuildNative`)

Reply:
hehe https://github.com/dotnet/runtime/pull/60701#discussion_r739375291

Reply:
Maybe all this is only needed for AOT  @radekdoulik ?